tests/testsupport/mqtt5_props.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2021
```

Comment:
This file and some other files in test have this license. Is this the correct license?

Reply:
No, it refers to the EPL but uProtocol uses the Apache Software License 2.0. It should be
```
# SPDX-FileCopyrightText: 2021 Contributors to the Eclipse Foundation
#
# See the NOTICE file(s) distributed with this work for additional
# information regarding copyright ownership.
#
# This program and the accompanying materials are made available under the
# terms of the Apache License Version 2.0 which is available at
# https://www.apache.org/licenses/LICENSE-2.0
#
# SPDX-License-Identifier: Apache-2.0
```

Reply:
@devkelley @sophokles73 for some of the files in tests folder, I have brought in test code from the paho.mqtt.python project, which is an eclipse project (https://github.com/eclipse/paho.mqtt.python/tree/master/tests). I have used the licensing that they used in some of their files. Can one of you let me know if there is a different way to reference that I'm using their code, or if it's even allowed to take code from other eclipse projects?

Reply:
Hmm, that is a good question. I am sure it depends partly on the EPL-2.0 license and what that stipulates. This FAQ seems to imply that it is okay to include as long as you include where you got the code from and the license (https://www.eclipse.org/legal/epl-2.0/faq.php). That being said it would probably be good to verify with someone from eclipse foundation